src/VisualStudio/Setup.Next/AssemblyRedirects.cs

```diff
@@ -6,6 +6,8 @@
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.EditorFeatures.Next.dll")]
 [assembly: ProvideRoslynBindingRedirection("Microsoft.VisualStudio.LanguageServices.Xaml.dll")]
 [assembly: ProvideRoslynBindingRedirection("Microsoft.VisualStudio.LanguageServices.Next.dll")]
+[assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.Remote.ServiceHub.dll")]
```

Comment:
this is for us to use devenv.exe.config in future (once change in core.d FI to our branch) instead of our own dll.config.